app/services/support_interface/add_course_choice_after_submission.rb

```diff
@@ -0,0 +1,23 @@
+module SupportInterface
+  class AddCourseChoiceAfterSubmission
+    attr_reader :application_form, :course_option, :send_to_provider
+
+    def initialize(application_form:, course_option:, send_to_provider: true)
+      @application_form = application_form
+      @course_option = course_option
+      @send_to_provider = send_to_provider
+    end
+
+    def call
+      application_choice = ApplicationChoice.create!(
+        application_form: application_form,
+        course_option: course_option,
+        status: 'awaiting_provider_decision',
```

Comment:
`SendApplicationToProvider` changes the state, so perhaps this should be `unsubmitted` to avoid errors.

Reply:
My bad, read https://github.com/DFE-Digital/apply-for-teacher-training/blob/90d463513e64b35934bb84e19ace5afbff606b7c/app/services/send_application_to_provider.rb#L15 as an `if` :+1: